### PR TITLE
Issue/downgrade prettytable dependency

### PR DESCRIPTION
**Pull request by arnaudsjs**
# Description

Downgrade the prettytable dependency to resolve dependency conflict.

# Self Check:

- [ ] ~~Attached issue to pull request~~
- [x] Changelog entry
- [x] Version number is bumped to dev version
- [ ] ~~Code is clear and sufficiently documented~~
- [ ] ~~Sufficient test cases (reproduces the bug/tests the requested feature)~~
- [ ] ~~Correct, in line with design~~
- [ ] ~~End user documentation is included or an issue is created for end-user documentation~~

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -40,7 +40,7 @@ oslo.log~=4.4
 oslo.serialization~=4.0
 oslo.utils~=4.7
 pbr~=5.5
-prettytable~=1.0
+prettytable~=0.7
```
Comment (bartv) [inline requirements.txt:43]: Why not remove it? Otherwise this error will keep popping up

Comment (arnaudsjs) [reply requirements.txt]: Because it remains a dependency. Anyway, I removed it from the list.

Comment (bartv) [reply requirements.txt]: But it is a transitive dependency? Which we now have an outstanding ticket to remove them from all modules?

**Review by bartv — APPROVED** [review]

**Comment by arnaudsjs** [comment]
Merged using 87238bf58bd6889d70fbaf5a3951c20834426680

**Pull request closed without merging**